changelogs/fragments/441-cloud-init_fix_removal_in_airgapped_environment.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfix:
+  - hosted_engine_setup - Fix cloud-init package removal in airgapped environment (https://github.com/oVirt/ovirt-ansible-collection/pull/441)
```

Comment:
Missing a newline at the end of the file

Reply:
Okay Will fix it tonight (in 10h)